src/core/processing/models/qgsprocessingmodelalgorithm.cpp

```diff
@@ -89,9 +89,6 @@ QVariantMap QgsProcessingModelAlgorithm::parametersForChildAlgorithm( const QgsP
   QVariantMap childParams;
   Q_FOREACH ( const QgsProcessingParameterDefinition *def, child.algorithm()->parameterDefinitions() )
   {
-    if ( def->flags() & QgsProcessingParameterDefinition::FlagHidden )
-      continue;
```

Comment:
@nyalldawson this fixes a crash for me as soon as I set the outputs to hidden (so I don't see them as parameters in the algorithm configuration widget). I am 0% sure this is the right approach to deal with this :D